chia-bls/fuzz/fuzz_targets/blspy-fidelity.rs

```diff
@@ -50,39 +55,56 @@ fuzz_target!(|data: &[u8]| {
 
         let idx = u32::from_be_bytes(<[u8; 4]>::try_from(&data[0..4]).unwrap());
         let rust_sk1 = rust_sk.derive_unhardened(idx);
-        let py_sk1 = aug.call_method1("derive_child_sk_unhardened",
-            PyTuple::new(py, &[py_sk, idx.to_object(py).as_ref(py)])).unwrap();
+        let py_sk1 = aug
+            .call_method1(
+                "derive_child_sk_unhardened",
+                PyTuple::new(py, &[py_sk, idx.to_object(py).as_ref(py)]),
+            )
+            .unwrap();
         assert_eq!(to_bytes(py_sk1), rust_sk1.to_bytes());
 
         let rust_pk1 = rust_pk.derive_unhardened(idx);
-        let py_pk1 = aug.call_method1("derive_child_pk_unhardened",
-            PyTuple::new(py, &[py_pk, idx.to_object(py).as_ref(py)])).unwrap();
+        let py_pk1 = aug
+            .call_method1(
+                "derive_child_pk_unhardened",
+                PyTuple::new(py, &[py_pk, idx.to_object(py).as_ref(py)]),
```

Comment:
are these `fmt` changes? it looks like perhaps this is to address clippy, but clippy still seems to be flagging this

Reply:
I'm going to be making a Clippy PR soon, this is to satisfy fmt since these weren't previously included in the workspace (and thus weren't checked by CI)